packages/cli/src/commands/start.ts

```diff
@@ -239,7 +239,7 @@ export class Start extends Command {
 
 		try {
 			// Start directly with the init of the database to improve startup time
-			const startDbInitPromise = Db.init().catch(async (error: Error) =>
+			await Db.init().catch(async (error: Error) =>
```

Comment:
The idea here was to allow db init to happen "async" while other things were happening; not a huge gain I believe, but are there any reasons to remove this behavior?

Reply:
it wasn't working as expected when i was testing it. it was somehow only waiting until the db connection was ready, but not until all the migrations were done executing. 
but, I can revert it, and test it again.

Reply:
I don't think it really provides any major gains, but it's just something to watch out for if we see people saying n8n is taking a bit longer to start. I don't believe we'll be seeing those issues frequently since it's a very simple thing.

Reply:
yeah. and since nodes are lazy loaded now, there shouldn't be much of a difference in startup times.